mmd_tools/operators/fileio.py

```diff
@@ -1206,3 +1201,96 @@ def execute(self, context):
             err_msg = traceback.format_exc()
             self.report({"ERROR"}, err_msg)
         return {"FINISHED"}
+
+
+# --- Drag and Drop Handlers ---
+
+
+class MMD_FH_PMX(bpy.types.FileHandler):
+    bl_idname = "MMD_FH_PMX"
+    bl_label = "MMD PMX/PMD Model"
+    bl_import_operator = "mmd_tools.import_model"
+    bl_file_extensions = ".pmx;.pmd"
+    bl_file_filter = "*.pmx;*.pmd"
+    bl_description = "Import MMD PMX/PMD Model by dropping into Blender"
+
+    @classmethod
+    def poll_drop(cls, context):
+        return context.area and context.area.type in {"VIEW_3D", "FILE_BROWSER"}
+
+    def import_drop(self, context, filepath=None, files=None, **kwargs):
+        if files:
+            for f in files:
+                bpy.ops.mmd_tools.import_model(filepath=f["name"])
+        elif filepath:
+            bpy.ops.mmd_tools.import_model(filepath=filepath)
+        return {"FINISHED"}
+
+
+class MMD_FH_VMD(bpy.types.FileHandler):
+    bl_idname = "MMD_FH_VMD"
+    bl_label = "MMD VMD Animation"
+    bl_import_operator = "mmd_tools.import_vmd"
+    bl_file_extensions = ".vmd"
+    bl_file_filter = "*.vmd"
+    bl_description = "Import MMD VMD Animation by dropping into Blender"
+
+    @classmethod
+    def poll_drop(cls, context):
+        return context.area and context.area.type in {"VIEW_3D", "FILE_BROWSER"}
+
+    def import_drop(self, context, filepath=None, files=None, **kwargs):
+        if files:
+            for f in files:
+                bpy.ops.mmd_tools.import_vmd(filepath=f["name"])
+        elif filepath:
+            bpy.ops.mmd_tools.import_vmd(filepath=filepath)
+        return {"FINISHED"}
+
+
+class MMD_FH_VPD(bpy.types.FileHandler):
+    bl_idname = "MMD_FH_VPD"
+    bl_label = "MMD VPD Pose"
+    bl_import_operator = "mmd_tools.import_vpd"
+    bl_file_extensions = ".vpd"
+    bl_file_filter = "*.vpd"
+    bl_description = "Import MMD VPD Pose by dropping into Blender"
+
+    @classmethod
+    def poll_drop(cls, context):
+        return context.area and context.area.type in {"VIEW_3D", "FILE_BROWSER"}
+
+    def import_drop(self, context, filepath=None, files=None, **kwargs):
+        if files:
+            for f in files:
+                bpy.ops.mmd_tools.import_vpd(filepath=f["name"])
+        elif filepath:
+            bpy.ops.mmd_tools.import_vpd(filepath=filepath)
+        return {"FINISHED"}
+
+
+# --- Keymap Registration ---
+
+
+addon_keymaps = []
+
+
+def register():
+    wm = bpy.context.window_manager
+    kc = wm.keyconfigs.addon
+    if not kc:
+        return
+
+    # Assign Ctrl + E as the hotkey for PMX export.
+    km = kc.keymaps.new(name="Screen", space_type="EMPTY")
+    kmi = km.keymap_items.new("mmd_tools.export_pmx", "E", "PRESS", ctrl=True, alt=False)
+    addon_keymaps.append((km, kmi))
```

Comment:
Are key bindings user-customizable?

Also, is this key binding safe?
https://www.google.com/search?q=Ctrl-E+site%3Adocs.blender.org